src/app.service.js

```diff
@@ -32,8 +32,14 @@ export class AppService {
     }
   }
 
-  getReadingTime(bodyElement, wpm) {
-    const mainContent = this.getMainContent(bodyElement);
+  getReadingTime(url, bodyElement, wpm) {
+    let mainContent = null;
+
+    if (url.includes("velog")) {
```

Comment:
```suggestion
    if (url.includes("velog.io")) {
```
url에 velog가 포함될 경우가 희박할 것 같지만 만약의 경우를 대비하여 조금 더 확실하게 `velog.io`로 특정지어주면 좋을 것 같습니다!

Reply:
해당 사항 반영하였습니다.